SNow v2 fix empty credentials error (#9308)
demisto/content: Packs/ServiceNow/ReleaseNotes/1_3_3.md

#### Integrations
##### ServiceNow v2

> - Fixed a bug where empty username or password in the instance configurations would result in an error.

Reviewer: ```suggestion
- Fixed a bug issue empty username or password in the instance configuration resulted in a non informative error.
```
